### ISSUE TITLE
NYU-V2 Depth dataset downloading

### ISSUE THREAD

**Issue by vuba-blog** [zhyever/Monocular-Depth-Estimation-Toolbox#61]
Thank you for your contribution,

I have a problem which might be a bit less related,
Currently I have troubles with downloading the NYU-V2 Depth dataset official test from: https://cs.nyu.edu/~silberman/datasets/nyu_depth_v2.html

it seems that the link is not working, I've found the training dataset from the BTS github, but canot find the official test.

Could you help?

Thank you so much.

**Comment by zhyever** [comment]
:D I'd like to help you. I will upload one.

**Comment by zhyever** [comment]
I have updated a new NYU download link. The test set is included in the zip file. Close this issue for now.
